etcdctl/README.md

```diff
@@ -565,6 +565,10 @@ Prints a humanized table of the member IDs, statuses, names, peer addresses, and
 
 ENDPOINT provides commands for querying individual endpoints.
 
+#### Options
+
+- cluster -- fetch and use all endpoints from the etcd cluster
```

Comment:
It says it's fetching it from the cluster though; if there's a pass-through proxy then the cluster wouldn't know about it. Would "fetch and use all endpoints from the etcd cluster member list" be OK?